### PR TITLE
[WIP] feat(radio/checkbox): add suffix for checkbox and radio

### PR DESCRIPTION
**Pull request by mohamedMok**
## I have read [the contributing guidelines](https://mozaic.adeo.cloud/Contributing/)

- [ ] Yes
- [ ] No

## Does this PR introduce a [breaking change](https://mozaic.adeo.cloud/Contributing/Developers/GitConventions/#breaking-changes-)?

- [ ] Yes
- [ ] No

## Describe the changes

GitHub issue number or Jira issue URL: https://github.com/adeo/mozaic-design-system/issues/665

### CODE REVIEW

File: packages/styles/components/_c.checkbox.scss [hunk 1]
```diff
@@ -12,6 +12,11 @@
     margin-left: magic-unit-rem(0.5, true);
   }
 
+  &__suffix-label {
```
Comment (tiloyi) [inline packages/styles/components/_c.checkbox.scss:15]: As the html element is already inside the label tag, we can just name this class `__suffix`.

File: packages/styles/components/_c.checkbox.scss [hunk 2]
```diff
@@ -12,6 +12,11 @@
...
+    color: $color-grey-600;
```
Comment (tiloyi) [inline packages/styles/components/_c.checkbox.scss:16]: It is preferable to use the variable that refers to the accessible text colour.:
~~$color-grey-600~~ => `$color-font-light`

File: packages/styles/components/_c.radio.scss [hunk 3]
```diff
@@ -12,6 +12,11 @@
     margin-left: magic-unit-rem(0.5, true);
   }
 
+  &__suffix-label {
+    color: $color-grey-600;
+    cursor: pointer;
+  }
```
Comment (tiloyi) [inline packages/styles/components/_c.radio.scss:18]: Same comment as above :)

**Comment by mohamedMok** [comment]
This PR is in WIP due to a design issue. @caqueste will create a new way to display suffix for form.

**Pull request closed without merging**